Goal: Do not hardcode the Python version in examples

Fixes: #380939?

Status wrt upstream: Forwarded upstream, will be in 3.0.26

Note: Probably other vendors would benefit from examples not hardcoding the Python version

Index: samba-3.0.25b/source/python/examples/tdbpack/tdbtimetrial.py
===================================================================
--- samba-3.0.25b.orig/source/python/examples/tdbpack/tdbtimetrial.py
+++ samba-3.0.25b/source/python/examples/tdbpack/tdbtimetrial.py
@@ -1,4 +1,4 @@
-#! /usr/bin/python2.2
+#! /usr/bin/python
 
 def run_trial():
     # import tdbutil
Index: samba-3.0.25b/source/python/examples/tdbpack/test_tdbpack.py
===================================================================
--- samba-3.0.25b.orig/source/python/examples/tdbpack/test_tdbpack.py
+++ samba-3.0.25b/source/python/examples/tdbpack/test_tdbpack.py
@@ -1,4 +1,4 @@
-#! /usr/bin/env python2.2
+#! /usr/bin/env python
 
 __doc__ = """test case for samba.tdbpack functions
 
